packages/ra-ui-materialui/src/list/BulkActionsToolbar.tsx

```diff
@@ -24,7 +19,7 @@ export const BulkActionsToolbar = (props: BulkActionsToolbarProps) => {
         className,
         ...rest
     } = props;
-    const { selectedIds = [], onUnselectItems } = useListContext(props);
+    const { selectedIds = [], onUnselectItems } = useListContext();
```

Comment:
Note that a standalone `<Datagrid>` with `bulkActionToolbars` not set to `false` will now fail loudly, wit han exception from `useListContext`. But it's better than before, because `<Datagrid>` only used to pass the `selectedIds` to the `BulkActionToolbar`, which didn't let the user select or unselect rows. I modified the doc to make it obvious that if you want bulk actions with a standalone Datagrid, you'll have to provide a context using `useList`.